<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPersonInfo</class>
 <widget class="QDialog" name="DialogPersonInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>758</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelPhoto">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelNo">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>工号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelName">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDepartment">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>部门：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPosition">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>职位：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRole">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>角色：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>